etna/articles/models.py

```diff
@@ -226,3 +229,166 @@ def latest_items(self) -> Tuple["ArticlePage"]:
+        "articles.ArticlePage",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        verbose_name=_("featured article"),
+    )
+
+    class Meta:
+        verbose_name = "Record Revealed article"
+        verbose_name_plural = "Record Revealed articles"
+
+    content_panels = BasePage.content_panels + [
+        FieldPanel("standfirst"),
+        MultiFieldPanel(
+            heading="Content Warning Options",
+            classname="collapsible collapsed",
+            children=[
+                FieldPanel("display_content_warning"),
+                FieldPanel("custom_warning_text"),
+            ],
+        ),
+        InlinePanel(
+            "gallery_images",
+            heading="Image Gallery",
+            label="Item",
+            min_num=1,
+            max_num=6,
+        ),
+        MultiFieldPanel(
+            heading="Body",
+            children=[
+                FieldPanel("record"),
+                FieldPanel("date_text"),
+                FieldPanel("about"),
+                FieldPanel("image_library_link"),
+                FieldPanel("print_on_demand_link"),
+            ],
+        ),
+        FieldPanel("featured_article"),
+        TopicalPageMixin.get_time_periods_inlinepanel(),
+        TopicalPageMixin.get_topics_inlinepanel(),
+    ]
+
+    promote_panels = MetadataPageMixin.promote_panels + TeaserImageMixin.promote_panels
+
+    @cached_property
+    def gallery_items(self):
+        """
+        Used to access the page's 'gallery_images' for output. Makes use of
+        Django's select_related() and prefetch_related() to efficiently
+        prefetch image and rendition data from the database.
+        """
+        return (
+            self.gallery_images.all()
+            .select_related("image")
+            .prefetch_related("image__renditions")
+        )
+
+
+class PageGalleryImage(Orderable):
+    page = ParentalKey(Page, on_delete=models.CASCADE, related_name="gallery_images")
+    image = models.ForeignKey(
+        "wagtailimages.Image", on_delete=models.SET_NULL, null=True, related_name="+"
+    )
+    is_sensitive = models.BooleanField(
+        verbose_name="Is this image sensitive?",
+        default=False,
+    )
+    alt_text = models.CharField(
+        verbose_name=_("alternative text"),
+        max_length=100,
+        help_text=mark_safe(
+            'Alternative (alt) text describes images when they fail to load, and is read aloud by assistive technologies. Use a maximum of 100 characters to describe your image. <a href="https://html.spec.whatwg.org/multipage/images.html#alt" target="_blank">Check the guidance for tips on writing alt text</a>.'
+        ),
+    )
+    caption = RichTextField(
+        help_text="An optional caption, which will be displayed directly below the image. This could be used for image sources or for other useful metadata.",
+        blank=True,
+    )
+    transcription_header = models.CharField(
```

Comment:
Note: the underscores in the field name is converted to a space form memory so we don't need to supply the verbose memory.

Reply:
Yes, this is on purpose. Whatever the project, I try to mark whatever simple strings I can for translation using `django.utils.translation.gettext_lazy`, as it takes very little effort to do as you go along.